packages/client/builtin/SlidevVideo.vue

```diff
@@ -1,11 +1,14 @@
 <script setup lang="ts">
 import { computed, onMounted, onUnmounted, ref, watch } from 'vue'
 import { useSlideContext } from '../context'
+import { useNav } from '../composables/useNav'
 
 const props = defineProps<{
   autoPlay?: boolean | 'once' | 'resume' | 'resumeOnce'
   autoPause?: 'slide' | 'click'
   autoReset?: 'slide' | 'click'
+  imgToPrint?: string
+  timeToPrint?: string | number | 'last'
```

Comment:
I am wonder if we could have a general props like `poster`: https://www.w3schools.com/tags/att_video_poster.asp

I am not so sure about props to be print specific - I would probably use JS to do it conditionally.